flask/app.py

```diff
@@ -157,6 +157,11 @@ class Flask(_PackageBoundObject):
     #: :class:`~flask.Response` for more information.
     response_class = Response
 
+    #: The class that is used for the Jinja environment.
+    #:
+    #: .. versionadded:: 0.11
```

Comment:
Not sure if this should be 0.11 or 1.0 - currently both versions are used, with 1.0 being more common but the current package version in `flask/__init__.py` being 0.11